helm/prometheus-rules/templates/platform/atlas/alerting-rules/grafana-postgresql.rules.yml

```diff
@@ -0,0 +1,44 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+    {{- if eq .Values.managementCluster.provider.flavor "vintage" }}
+    cluster_type: "management_cluster"
+    {{- end }}
+  name: grafana-postgresql.rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+  - name: grafana-postgresql
+    rules:
+    - alert: GrafanaPostgresqlDown
+      annotations:
+        description: '{{`The grafana-postgresql cluster has one or more unhealthy instances.`}}'
+        runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/grafana-down/
+      expr: count(up{container="postgres", namespace="monitoring"}) by (container, cluster_id, installation, pipeline, provider) < 2
+      for: 5m
+      labels:
+        area: platform
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_status_updating: "true"
+        cancel_if_outside_working_hours: "true"
+        severity: page
+        team: atlas
+        topic: observability
+    - alert: GrafanaPostgresqlVolumeLowSpace
```

Comment:
Do we want to create a specific alert for this component or do we want to rely on https://github.com/giantswarm/prometheus-rules/blob/cc79f1b4205ad8c1020583a984c59ea36063bd94/helm/prometheus-rules/templates/platform/atlas/alerting-rules/storage.rules.yml#L10?

I don't really like to create alert based on symptoms but I don't know if we can do much for now until we have better alerts anyway

Reply:
I guess you're right. I'll cut this specific alert and add the grafana-postgresql to the already existing one.